Update github-vs-sourcegraph.md (#44435)
sourcegraph/sourcegraph: doc/getting-started/github-vs-sourcegraph.md

**Forks** are included in the index, but they are subject to the same limitations as other repositories, so not all forks are indexed.

> GitHub code search supports searching issues and pull requests along with the codebase. In addition to searching your private code, GitHub has indexed over 7 million public GitHub repositories which are also searchable.

Author: ```suggestion
GitHub code search supports searching across issues, pull requests, commits, packages, discussions, and more. In addition to searching your private code, GitHub has indexed over 7 million public GitHub repositories which are also searchable.
```

> GitHub code search supports searching across issues, pull requests, commits, packages, discussions, and more. In addition to searching your private code, GitHub has indexed over 7 million public GitHub repositories which are also searchable.

Author: ```suggestion
GitHub code search supports searching across issues, pull requests, and discussions. In addition to searching your private code, GitHub has indexed over 7 million public GitHub repositories which are also searchable.
```

   <td>2.8 million 
   </td>
  </tr>
  <tr>

>    <td><strong>Search across issues and pull requests</strong>

Author: ```suggestion
   <td><strong>Search across issues, pull requests, commits, packages, discussions, and more.</strong>
```

>    <td><strong>Search across issues, pull requests, commits, packages, discussions, and more.</strong>

Author: ```suggestion
   <td><strong>Search across issues, pull requests, and discussions. </strong>
```
